scvi/module/_vae.py

```diff
@@ -96,6 +98,7 @@ def __init__(
         use_batch_norm: Literal["encoder", "decoder", "none", "both"] = "both",
         use_layer_norm: Literal["encoder", "decoder", "none", "both"] = "none",
         use_observed_lib_size: bool = True,
+        var_activation: Optional[Callable] = torch.exp,
```

Comment:
Actually let's make this None, and use torch.exp if None. Be sure to add in docstring, If `None`, defaults to :func:`~torch.exp`.

Reply:
Done!